modules/gdscript/gdscript_compiler.cpp

```diff
@@ -2123,15 +2123,21 @@ void GDScriptCompiler::_make_scripts(GDScript *p_script, const GDScriptParser::C
 		StringName name = p_class->subclasses[i]->name;
 
 		Ref<GDScript> subclass;
+		String fully_qualified_name = p_script->fully_qualified_name + "::" + name;
```

Comment:
You might need to rebase on the master branch, as those changes should not be part of this PR since they were already merged.

Reply:
But this line isnt part of the PR, in master subclass->fully_qualified_name is initiated directly, in this commit i moved it to a local variable to reuse the value instead of calculating it twice.